game/theater/controlpoint.py

```diff
@@ -120,6 +120,11 @@ class PresetLocations:
 
     #: Locations of stationary armor groups.
     armor_groups: List[PointWithHeading] = field(default_factory=list)
+    armor_shorad_groups: List[PointWithHeading] = field(default_factory=list)
```

Comment:
This should just be `armor_groups`, and the type should change to `ArmorGroupLocation`, a new class that includes the details. Currently it'd just be the location and a boolean for SHORADS inclusion, but in the next PR it'd add an enum for the group type. Presumably this doesn't just end at light/heavy, and will eventually include everything in `GroundUnitRole`. `PresetLocations` shouldn't be growing to include the cross-product of all armor group properties.

Reply:
Sounds good, will do it.